docs/how-to/deploy_sdcore_user_plane_in_dpdk_mode.md

```diff
@@ -0,0 +1,125 @@
+# Deploy SD-Core User Plane in DPDK mode
+
+This guide covers how to deploy the User Plane Function (UPF) in DPDK mode using 
+the `sdcore-user-plane` Juju bundle.
+
+## Requirements
```

Comment:
Are we planning to add the steps to prepare the whole environment in a single host similar to mastering guide ?

Reply:
As this is a howto just for enabling DPDK, we will not be guiding the users to that level.